src/main.rs

```diff
@@ -61,7 +75,9 @@ fn main() -> anyhow::Result<()> {
         Esp32TlsServerConfig::new(cert, key.as_ptr(), key.len() as u32)
     };
 
-    let repr = RobotRepresentation::WithRegistry(ComponentRegistry::default());
+    let mut registry = Box::new(ComponentRegistry::default());
```

Comment:
this will change the signature of WithRegistry is it intentional?

Reply:
It is intentional. This is actually needed to catch viamrobotics/micro-rdk-robot-template up with the micro-rdk changes here: https://github.com/viamrobotics/micro-rdk/commit/01cab144489d56b41555fbf8cee7f871303b1725#diff-e20dfa7bb109ea82f866b58aea13fc2bf617a1e69b5b2894c217faa6e3eab479R6